run-directive/action.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Directive
+run-name: test
+
+inputs:
+  hash:
+    description: 'Github commit hash to checkout'
+    required: false
+    default: ''
```

Comment:
Does this default correctly fallback to latest?

Reply:
I'll look more into this. The wording for [the action](https://github.com/Brightspace/third-party-actions/tree/actions/checkout?tab=readme-ov-file#usage) is not 100% clear.

> The branch, tag or SHA to checkout. When checking out the repository that
    triggered a workflow, this defaults to the reference or SHA for that event.
    Otherwise, uses the default branch.

Other inputs have Default and it's value listed, but this one does not.

I shall figure it out!

Reply:
Running this locally, it was checking out master this input was left blank. This is what I used for the output in the pr description.

I also tested with a the previous commit sha and it checked that out.